waywiser_command_control/package.xml

```diff
@@ -21,6 +21,8 @@
   <depend>std_msgs</depend>
```

Comment:
I am wondering whether waywiser_command_control is a good name, isn't it more like waywiser_cmd_vel_safety? To my understanding it is the vehicle side of processing cmd_vel safely.

Reply:
Package renamed to waywiser_twist_safety.